src/quote/swap/adapters/static/swap-data-config.ts

```diff
@@ -137,6 +137,29 @@ export const SwapDataConfig: Readonly<{
         },
       },
     },
+    'icETH': {
+      '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2': {
+        contract: 'ExchangeIssuanceLeveraged',
+        swapDataDebtForCollateral: {
+          path: [
+            '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
+            '0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84',
+          ],
+          fees: [],
+          exchange: 4,
+          pool: '0xDC24316b9AE028F1497c275EB9192a3Ea0f67022',
+        },
+        swapDataInputToken: {
+          path: [
+            '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
+            '0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84',
+          ],
+          fees: [],
+          exchange: 4,
+          pool: '0xDC24316b9AE028F1497c275EB9192a3Ea0f67022',
+        },
+      },
+    },
```

Comment:
Right now only redeeming to ETH/WETH is supported. What else should be supported? stETH? USDC?